projects/view/client/projectProfile/projectProfile.html

```diff
@@ -57,8 +77,99 @@
         <p>{{ description }}</p>
       </div>
     </div>
+</div>
+<!-- TODO wrap content with class=row divs where proper -->
+<!-- START OUT OF SCOPE FOR RELEASE 0.2 -->
+<!-- START Project Positions Content (aka Requests) -->
+<div id="project-positions" class="col-md-12">
+  <div id="project-positions-names">
+    <h2 id="project-position-header">Opening Positions</h2>
+    <span id="project-position-main-description"> If you have the right skillset and expereince, you are more than welcome to help out.</span>
+    <div id="project-position-names-seperator" class="col-md-6">
+      <div id="project-position-lists" class="panel-body">
```

Comment:
The `panel-body` class needs to be contained within a `panel` element. Check [Bootstrap panel](http://getbootstrap.com/components/#panels) docs.